framework/source/class/qx/theme/indigo/Appearance.js

```diff
@@ -183,8 +183,8 @@ qx.Theme.define("qx.theme.indigo.Appearance",
     "list-search-highlight" : {
       style : function(states) {
         return {
-          color      : '#FF0000',
-          fontWeight : 'bold'
+          text-decoration-style: dotted,
```

Comment:
Does this work, as javascript? Don't you have to put it into quotation marks?

Reply:
I mean: wouldn't `text-decoration-style` be interpreted as a subtraction of unknown variables?

Reply:
Of course not ... fixed.